### PR TITLE
Fixes #32371 - CLI changes for importing roles & variables

### PR DESCRIPTION
**Pull request by shiramax**
depend on: 
https://github.com/theforeman/foreman_ansible/pull/396

### CODE REVIEW

**Comment by xprazak2** [comment]
Works nicely, should this have its own Redmine issue rather than reusing the one for API?

**Comment by shiramax** [comment]
@xprazak2 done

File: lib/hammer_cli_foreman_ansible/ansible_roles.rb [hunk 1]
```diff
@@ -87,7 +96,13 @@ class FetchCommand < HammerCLIForeman::Command
 
       output do
         collection :ansible_roles, _('Ansible roles available to be imported') do
-          field :name, nil
+          field :name, _("Role Name")
+          field :role_action, _("Action")
+          field :variables, _("Variables")
+          field :hosts_count, ("Hosts count")
+          field :hostgroup_count, ("Hostgroup_count")
```
Comment (xprazak2) [inline lib/hammer_cli_foreman_ansible/ansible_roles.rb:103]: Super-small nitpick: should this be `Hostgroup count`?

**Comment by xprazak2** [comment]
Thanks!

**Pull request merged**